modules/node_modules/@ncigdc/uikit/Dropdown.js

```diff
@@ -35,7 +37,7 @@ const Dropdown = ({
         onMouseUp={mouseUpHandler}
         onClick={() => setTimeout(() => setActive(false))}
       >
-        {Children.map(children, child =>
+        {Children.map(_.filter(children, React.isValidElement), child =>
```

Comment:
Appreciate the tip but this is intentional  

Using the native filter naievely would result in `Cannot read property 'filter' of undefined` when there are no children (children is undefined), or when there is a single string/number/react element.

The code becomes rather unnecessarily verbose, for the benefit of not using a function in a library we're using anyway. Cost/benefit doesn't work out